src/arch/wasm32/black_box.s

```diff
@@ -0,0 +1,13 @@
+	.text
```

Comment:
Seems to be the wasm LLVM assembler isn't a first class tool out there. I couldn't find any documentation on the syntax and I just figured it out empirically. It was really painful because this assembler acts unpredictably.

Unfortunately, there is no source to link to.

Reply:
Ah ok, perhaps having this comment on the PR is enough in that case :)